heron/spi/src/java/com/twitter/heron/spi/utils/TopologyUtils.java

```diff
@@ -50,7 +50,7 @@ private TopologyUtils() {
 
```

Comment:
`verifyTopology` should really be fixed to return `void` and throw a better exception that `RuntimeException` (with a better message) but that's a fix for another PR...